docs/zh-CN/docs/sql-manual/sql-functions/json-functions/json_valid.md

```diff
@@ -0,0 +1,75 @@
+---
+{
+    "title": "json_valid",
+    "language": "zh-CN"
+}
+---
+
+<!-- 
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## json_valid
+### description
+
+json_valid 函数返回0或1以表明是否为有效的JSON, 如果参数是NULL则返回NULL。
+
+#### Syntax
+
+`JSONB json_valid(VARCHAR json_str)`
+
+### example
+
+1. 正常JSON字符串
+
+```
+MySQL > SELECT json_valid('{"k1":"v31","k2":300}');
++-------------------------------------+
+| json_valid('{"k1":"v31","k2":300}') |
++-------------------------------------+
+|                                   1 |
++-------------------------------------+
+1 row in set (0.02 sec)
+```
+
+2. 无效的JSON字符串
+
+```
+MySQL > SELECT json_valid('invalid json');
++----------------------------+
+| json_valid('invalid json') |
++----------------------------+
+|                          0 |
++----------------------------+
+1 row in set (0.02 sec)
+```
+
+3. NULL参数
+
+```
+MySQL > select json_valid(NULL);
++------------------+
+| json_valid(NULL) |
++------------------+
+|             NULL |
++------------------+
+1 row in set (0.02 sec)
+```
+
+### keywords
+JSONB, JSON, json_valid
```

Comment:
```suggestion
JSON, VALID, JSON_VALID
```
Why there is JsonB? seems the argument is always string.

Reply:
let me fix it